public/app/features/scopes/testUtils.tsx

```diff
@@ -464,18 +463,14 @@ export function buildTestScene(overrides: Partial<DashboardScene> = {}) {
 }
 
 export function renderDashboard(dashboardScene: DashboardScene) {
-  const store = configureStore();
   const chrome = new AppChromeService();
```

Comment:
The `AppChromeService` is not needed in this case. The rationale for creating a wrapper was to pass this instance to it in order to ensure we get a chromeless version.

Reply:
Cool, I'll remove it 👍

Reply:
(FYI whilst we're on the topic of router + tests, it seems you don't need the `historyOptions` config to set `initialEntries`, tests pass without it)

Reply:
TBH I'm a little bit intrigued what changed since I wrote those tests. Previously the custom app chrome service instance and the initial history entries were needed for the tests to pass 🤔 .

PS: I just tested and I can confirm the `initialEntries` are not needed anymore.